EKF/ekf.h

```diff
@@ -598,10 +598,7 @@ class Ekf : public EstimatorInterface
 	void checkForStuckRange();
 
 	// return the square of two floating point numbers - used in auto coded sections
-	inline float sq(float var)
-	{
-		return var * var;
-	}
+	static constexpr float sq(float var) { return var * var; }
```

Comment:
Why has the inline attrribute been removed function? inlining has been used in the past in the past to preserve speed of autocode. What effect will this change have on the compiled code? My intention was that the compiled code usign this function would be the same as if I had used var*var instead.

Reply:
Adding constexpr implies inline, but also means it can be fully evaluated at compile time (if the input is known at compile time).

The mathlib helper functions for degrees <-> radians, constrain, min, max are all constexpr as well. Overall that means statements like `sq(math::radians(10.0f))` really do turn into `0.0304617f` at compile time. In simple cases this might happen if they were all inline as well, but it's dependant on compiler optimization level, function size, etc.